src/coreclr/jit/emitriscv64.cpp

```diff
@@ -2854,7 +2847,12 @@ size_t emitter::emitOutputInstr(insGroup* ig, instrDesc* id, BYTE** dp)
             assert(!"JitBreakEmitOutputInstr reached");
         }
     }
-#endif
+#else  // !DEBUG
+    if (emitComp->opts.disAsm)
+    {
+        emitDispIns(id, false, false, true, 0, *dp, (dstRW - odstRW), ig);
+    }
+#endif // !DEBUG
```

Comment:
Why can't you choose whether to display offsets (emitComp->opts.disDiffable) like in debug?

Reply:
Both [x86](https://github.com/dotnet/runtime/blob/main/src/coreclr/jit/emitxarch.cpp#L17515) and [arm64](https://github.com/dotnet/runtime/blob/88b5e3d4b77dd8238331ade1b31ac8ddc62f22f7/src/coreclr/jit/emitarm64.cpp#L14361) doesn't allow user to display offsets in release mode. I wanted my implementation to behave similarly to those two